zmq-lwt/src/zmq_lwt.ml

```diff
@@ -0,0 +1,82 @@
+let (>>=) = Lwt.(>>=)
```

Comment:
Opening `Lwt.Infix` might be the way to go here.

Reply:
I think this will go away in the functorization anyway.